paddle/fluid/distributed/collective/custom_ccl_tools.cc

```diff
@@ -29,7 +29,7 @@ phi::ccl::CCLReduceOp ToCustomCCLRedType(ReduceOp reduction) {
   PADDLE_ENFORCE_EQ(
       it != red_type.end(),
       true,
-      platform::errors::InvalidArgument("Invalid hccl reduction. "
+      platform::errors::InvalidArgument("Invalid CustomCCL reduction. "
```

Comment:
I have some questions on this block of code, is this only work for hccl now or any other custom device? if the 2nd condition, is it reasonable to check reduce op in [min max sum prod]? @qili93 @ronny1996

Reply:
this block work for custom device